### PR TITLE
run-regression-test: fix a bug that MailNotifier doesn't work

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: lib/groonga-query-log/command/run-regression-test.rb [hunk 1]
```diff
@@ -73,6 +73,7 @@ def initialize
           smtp_auth_password: nil,
           smtp_starttls: false,
           smtp_port: 25,
+          working_directory: @working_directory,
```
Comment (kou) [inline lib/groonga-query-log/command/run-regression-test.rb]: How about `path: "#{@working_directory}/results"`?

**Comment by komainu8** [comment]
@kou Thank you for your comment.
I've fixed it.

File: test/command/test-run-regression-test.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```
Comment (kou) [inline test/command/test-run-regression-test.rb]: Could you remove this?

File: lib/groonga-query-log/command/run-regression-test.rb [hunk 3]
```diff
@@ -678,7 +679,7 @@ def initialize(success, elapsed_time, options)
           @success = success
           @elapsed_time = elapsed_time
           @options = options
-          @path = @options[:path] || "results"
+          @path = @options[:path]
```
Comment (kou) [inline lib/groonga-query-log/command/run-regression-test.rb]: Could you keep this  default value?

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

**Pull request merged**